assets/app/view/spreadsheet.rb

```diff
@@ -2,10 +2,14 @@
 
 require 'view/stock_market'
 require 'lib/color'
+require 'view/link'
+require 'lib/storage'
 
 module View
   class Spreadsheet < Snabberb::Component
     needs :game
+    needs :spreadsheet_sort_by, default: Lib::Storage['spreadsheet_sort_by'] || 'ID', store: true
```

Comment:
i made this framework and didn't even know you could do this lol.. probably safer to default to nil and then to reset it in the render

Reply:
agreed.